lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -194,6 +194,20 @@ module.exports = [
       }
     }
   }, {
+    initialUrl: 'https://www.chromestatus.com/features',
```

Comment:
Up until now we've entirely controlled the expectations here which can be run locally, can we not move this to belong with other external sites?